softwerkskammer/lib/mailsender/mailsenderService.js

```diff
@@ -166,14 +166,14 @@ module.exports = {
     });
   },
 
-  sendMailToContactPersonsOfGroup: function (groupName, message, callback) {
+  sendMailToContactPersonsOfGroup: function (groupId, message, callback) {
```

Comment:
As far as I remember we do not have a test for this one where the group does not exist.